src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs

```diff
@@ -2067,6 +2067,13 @@ private void CheckForEqualityAndGetHashCode(DiagnosticBag diagnostics)
                 return;
             }
 
+            if (IsRecord)
+            {
+                // For records the warnings reported below are simply going to eco record specific errors,
```

Comment:
>eco [](start = 79, length = 3)

Typo?

Reply:
> Typo?

Yes, thanks

---
In reply to: [464556651](https://github.com/dotnet/roslyn/pull/46497#discussion_r464556651) [](ancestors = 464556651)